Tribler/Core/Session.py

```diff
@@ -625,7 +625,14 @@ def search_remote_torrents(self, keywords):
         :param keywords: The given keywords.
         :return: The number of requests made.
         """
-        return self.lm.torrent_search_manager.search(keywords)
+        return self.lm.search_manager.search_for_torrents(keywords)
+
+    def search_remote_channels(self, keywords):
+        """
+        Searches for remote channels through AllChannelCommunity with the given keywords.
+        :param keywords: The given keywords.
+        """
+        self.lm.search_manager.search_for_channels(keywords)
```

Comment:
@NielsZeilemaker I removed the `OperationNotEnabledByConfigurationException()` here because in `torrent_search` and `channel_search`, only one community is enabled, but the test uses `GuiUtility` to search for results, which always calls both `search_remote_torrents` and `search_remote_channels`, so it will fail if we add this check.